src/main/java/com/kyraltre/tretackshop/block/TackShopBlocks.java

```diff
@@ -25,61 +25,61 @@ public class TackShopBlocks {
-            () -> new PumpkinTrophy(BlockBehaviour.Properties.copy(Blocks.IRON_BLOCK).noOcclusion().strength(1.0F)
+            () -> new PumpkinTrophy(AbstractBlock.Properties.from(Blocks.IRON_BLOCK).notSolid().hardnessAndResistance(1.0F)
             ), TackShopCreativeModTab.TREAWARD_TAB);
 
     public static final RegistryObject<Block> PUMPKIN_RESERVE = registerBlock("pumpkin_reserve",
-            () -> new PumpkinTrophy(BlockBehaviour.Properties.copy(Blocks.IRON_BLOCK).noOcclusion().strength(1.0F)
+            () -> new PumpkinTrophy(AbstractBlock.Properties.from(Blocks.IRON_BLOCK).notSolid().hardnessAndResistance(1.0F)
             ), TackShopCreativeModTab.TREAWARD_TAB);
 
     public static final RegistryObject<Block> SNOWFLAKE_GRAND = registerBlock("snowflake_grand", () ->
-            new SnowflakeTrophy(BlockBehaviour.Properties.copy(Blocks.IRON_BLOCK).noOcclusion().strength(1.0F)
+            new SnowflakeTrophy(AbstractBlock.Properties.from(Blocks.IRON_BLOCK).notSolid().hardnessAndResistance(1.0F)
             ), TackShopCreativeModTab.TREAWARD_TAB);
     public static final RegistryObject<Block> SNOWFLAKE_RESERVE = registerBlock("snowflake_reserve",
-            () -> new SnowflakeTrophy(BlockBehaviour.Properties.copy(Blocks.IRON_BLOCK).noOcclusion().strength(1.0F)
+            () -> new SnowflakeTrophy(AbstractBlock.Properties.from(Blocks.IRON_BLOCK).notSolid().hardnessAndResistance(1.0F)
             ), TackShopCreativeModTab.TREAWARD_TAB);
     public static final RegistryObject<Block> HEART_GRAND = registerBlock("heart_grand", () ->
-            new HeartTrophy(BlockBehaviour.Properties.copy(Blocks.IRON_BLOCK).requiresCorrectToolForDrops().noOcclusion().strength(1.0F)
+            new HeartTrophy(AbstractBlock.Properties.from(Blocks.IRON_BLOCK).harvestLevel(1).notSolid().hardnessAndResistance(1.0F)
             ), TackShopCreativeModTab.TREAWARD_TAB);
     public static final RegistryObject<Block> HEART_RESERVE = registerBlock("heart_reserve",
-            () -> new HeartTrophy(BlockBehaviour.Properties.copy(Blocks.IRON_BLOCK).requiresCorrectToolForDrops().noOcclusion().strength(1.0F)
+            () -> new HeartTrophy(AbstractBlock.Properties.from(Blocks.IRON_BLOCK).notSolid().hardnessAndResistance(1.0F)
             ), TackShopCreativeModTab.TREAWARD_TAB);
 
     public static final RegistryObject<Block> CLOVER_GRAND = registerBlock("clover_grand", () ->
-            new CloverTrophy(BlockBehaviour.Properties.copy(Blocks.IRON_BLOCK).requiresCorrectToolForDrops().noOcclusion().strength(1.0F)
+            new CloverTrophy(AbstractBlock.Properties.from(Blocks.IRON_BLOCK).notSolid().hardnessAndResistance(1.0F)
             ), TackShopCreativeModTab.TREAWARD_TAB);
     public static final RegistryObject<Block> CLOVER_RESERVE = registerBlock("clover_reserve",
-            () -> new CloverTrophy(BlockBehaviour.Properties.copy(Blocks.IRON_BLOCK).requiresCorrectToolForDrops().noOcclusion().strength(1.0F)
```

Comment:
These where I removed the `.requiresCorrectToolForDrops()` was not translated to the proper `.harvestLevel()` and `.harvestTool()` builder methods, you should fix that.